spec/shoot_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'shoot'
+
+describe ModifiedArray do
+
+  it "returns 1 when called on array containing integer 1 with no arguments" do
+    array = ModifiedArray.new
+    array << 1
+    expect( array.shoot { |accumulator, element| element }).to eq 1
```

Comment:
Unused block argument - `accumulator`. If it's necessary, use `_` or `_accumulator` as an argument name to indicate that it won't be used.<br>Space inside parentheses detected.